Bio/SeqUtils/MeltingTemp.py

```diff
@@ -567,7 +567,7 @@ def salt_correction(Na=0, K=0, Tris=0, Mg=0, dNTPs=0, method=1, seq=None):
         corr = 0.368 * (len(seq) - 1) * math.log(mon)
     if method == 6:
         corr = (
-            (4.29 * SeqUtils.GC(seq) / 100 - 3.95) * 1e-5 * math.log(mon)
+            (4.29 * SeqUtils.gc_content(seq) - 3.95) * 1e-5 * math.log(mon)
```

Comment:
Should this be using ``ignore_ambiguous=False`` to preserve the old behaviour? Or is this change OK?

Either way, it tells me the salt correction probably did not have a unit test with any ambiguous bases in it.

Reply:
> Should this be using `ignore_ambiguous=False` to preserve the old behavior? Or is this change OK?
> 
> Either way, it tells me the salt correction probably did not have a unit test with any ambiguous bases in it.

I was only basing it in the test results, but will admit I didn't investigate deeply how the function works is this particular case. Looking more closely I don't think there is a way to reproduce the behavior of `GC()` in this particular case with `gc_content()`(in it's current form). 

I say that because the `_check()` function in this module does not remove the ambiguous nucleotides for the "Tm_GC" method, so the original behavior would be to count them in the length of the sequence but not in the GC percentage. I apologize for not paying more attention to this.

I see a few options on how to proceed:

1. Let this unchanged with the old GC() function and let someone more familiar with the code fix it in the future.
2. Add a new option in `gc_content` allowing this specific behaviour (maybe replace `ignore_ambiguous` with an argument called `ambiguous` with 3 different values, `{'count', 'ignore', 'remove'}` for example?)
3. Replace this line with some code counting "GCSgcs" % in this sequence not using the `gc_content` or `GC` functions.

I prefer option 2, but would like to know what you think is best in this case.